tarefas 1,2,3,4,5/myscript.js

```diff
@@ -1,35 +1,30 @@
+const bgescuro = document.querySelector("#imgescura");
+const imgescura = document.querySelector("#escuro");
+const icones = document.querySelectorAll(".icones");
+const cards = document.querySelectorAll(".card");
+const barra = document.querySelector(".barra");
+const botao = document.querySelector("button");
+const textos = document.querySelectorAll("a, h1, h2");
+const bloco = document.querySelector(".bloco");
+const bloquinho = document.querySelector(".bloquinho");
 
 function escurecer() {
-	let i;
-	const bgescuro = document.querySelector("#imgescura");
-	const imgescura = document.querySelector("#escuro");
-	const icones = document.querySelectorAll(".icones");
-	const cards = document.querySelectorAll(".card");
-	const barra = document.querySelector(".barra");
-	const botao = document.querySelector("button");
-	const textos = document.querySelectorAll("a, h1, h2");
-	const bloco = document.querySelector(".bloco");
-	const bloquinho = document.querySelector(".bloquinho");
-	let dentro = botao.innerHTML;
+	const dentro = botao.innerHTML;
 
-	for(i = 0; i < textos.length; i++){
+	for(let i = 0; i < textos.length; i++){
 		textos[i].classList.toggle("subitem");
 	}
 
-	if(dentro == "LIGHT") {botao.innerHTML = "DARK";}
-	else {botao.innerHTML = "LIGHT";}
+	botao.innerHTML = dentro === "LIGHT" ? "DARK" : "LIGHT";
 
 	botao.classList.toggle("botaoclaro");
 	bloquinho.classList.toggle("bloquinhoescuro");
 	bloco.classList.toggle("blocoescuro");
 	bgescuro.classList.toggle("sumir");
 	imgescura.classList.toggle("sumir");
-	for(i = 0; i < icones.length; i++){
-		icones[i].classList.toggle("escurinho");
-	}
+	icones.forEach((icone) => icone.classList.toggle("escurinho"));

```

Comment:
Refactor: substitui loop for por um método de array forEach como visto na aula 6, não é necessário mas sugiro que repita isso nos demais loops.